src/wcag-taxonomy-provider.test.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as Sarif from './sarif/sarif-2.1.2';
+import { getWcagTaxonomy } from './wcag-taxonomy-provider';
+
+describe('WCAGTaxonomyProvider', () => {
+    it('creates a Sarif.ToolComponent object with WCAG property information', () => {
+        const sortedTagsStub: string[] = [];
+        const expectedResultsStub: Sarif.ToolComponent = {
+            name: 'WCAG',
+            fullName: 'Web Content Accessibility Guidelines (WCAG) 2.1',
+            organization: 'W3C',
+            informationUri: 'https://www.w3.org/TR/WCAG21',
+            version: '2.1',
+            guid: '',
+            isComprehensive: true,
+            taxa: [],
+        };
+        const actualResults: Sarif.ToolComponent = getWcagTaxonomy(
+            sortedTagsStub,
+        );
+
+        expect(actualResults).toEqual(expectedResultsStub);
+    });
+
+    it('creates a Sarif.ToolComponent object with taxa properties based on axeTagsToWcagLinkData', () => {
+        const sortedTagsStub: string[] = ['wcag111', 'wcag121'];
+        const expectedResults = [
+            {
+                id: 'wcag111',
+                name: 'WCAG 1.1.1',
```

Comment:
It feels a little uncomfortable to me that this is testing the implementation of axeTagsToWcagLinkData; consider instead injecting the link data into getWcagTaxonomy (maybe as a second parameter) so you can use stub data in the tests

Reply:
(wcag-link-data-processor.test.ts does a good job of this already :) )